src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCacheEntry.java

```diff
@@ -79,9 +78,14 @@ public DefaultCacheEntry(
         this.responsePool = responsePool;
     }
 
-    public BufferPool getResponsePool()
+    public void setSubscribers(int numberOfSubscribers)
     {
-        return responsePool;
+        this.subscribers += numberOfSubscribers;
+    }
```

Comment:
Should this method be called `addSubscribers` since the implementation is `+=` ?

Reply:
I think it is a bit tricky since we are also passing a negative value to decrement.